### PR TITLE
Fix import and peers set

### PR DESCRIPTION
**Pull request by vcastellm**
After subsequent peer `Set`, peers were empty due to a missing `Add` to the peers map.

Also add support for modules and fix `logrus` import

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module github.com/distribworks/gcgrpcpool
```
Comment (charithe) [inline go.mod]: This only applies to your fork :smile:

**Comment by charithe** [comment]
Thanks for the PR. Unfortunately, the module files are for your fork so I can't merge them into this repo. 

Out of interest, do you actually use this library in an application?

File: go.mod (hunk 1 above)
Comment (vcastellm) [reply go.mod]: Yup sorry, fixed

**Comment by vcastellm** [comment]
@charithe fixed the module path.

Yes, I'm using it for https://dkron.io/ and working great with these fixes so far.

**Comment by charithe** [comment]
Thanks. It's good to know you find it useful.

**Pull request merged**